setup.cfg

```diff
@@ -50,6 +50,7 @@ install_requires =
     pycryptodomex  # for EncryptedKeyring backend in keyrings.alt
     pydantic >= 1.9.0
     pynwb >= 1.0.3,!=1.1.0
+    nwbinspector @ git+https://github.com/NeurodataWithoutBorders/nwbinspector@dev
```

Comment:
Please release a new version of nwbinspector so that this PR can be merged (and so that a conda-forge feedstock can be created).

Reply:
@jwodder Hey there - I think we might need 1 or 2 bug fixes plus a new feature. I'm on vaca right now, but I'll be back on Thursday to try to push this out before the end of the week

Reply:
@jwodder Pushed new version